backend/src/test/java/de/bund/digitalservice/ris/norms/utils/TestZipHelper.java

```diff
@@ -0,0 +1,83 @@
+package de.bund.digitalservice.ris.norms.utils;
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * Helper class to create test ZIP files with various structures
+ * for use in unit tests verifying ZIP extraction and validation.
+ * <p>
+ * Supports creating ZIPs with path traversal entries, folder entries,
+ * and archives with too many files.
+ */
+public class TestZipHelper {
```

Comment:
is there a reason for this class to exist and to create the files on the fly instead of adding the 3 files to the resources folder? This seems a lot more fragile than having the files laying around, especially as i do not expect anything about them to ever need to change.